rkt/api_service.go

```diff
@@ -82,6 +82,14 @@ func (s *v1AlphaAPIServer) GetInfo(context.Context, *v1alpha.GetInfoRequest) (*v
 			RktVersion:  version.Version,
 			AppcVersion: schema.AppContainerVersion.String(),
 			ApiVersion:  supportedAPIVersion,
+			GlobalFlags: &v1alpha.GlobalFlags{
+				Dir:                globalFlags.Dir,
```

Comment:
Use `getDataDir()` instead.

globalFlags.Dir is used only when it was actually modified via `--dir` flag. Otherwise it tries to get it from config and falls back to the default value `/var/lib/rkt`.